.gitignore

```diff
@@ -1,3 +1,19 @@
 /target
 /vendor/boost
 /touchHLE_sandbox
+
+# Android
+
+*.iml
+.gradle
+/local.properties
+.idea/
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
+*.so
```

Comment:
Huh, why is there a wildcard ignore for `.so` files, surely they won't be dumped at the base of the repository?

Reply:
i followed SDL .gitgignore https://github.com/libsdl-org/SDL/blob/release-2.26.4/.gitignore#L20

Otherwise generated .so are in `android/app/src/main/jniLibs/arm64-v8a/libtouchHLE.so`, so i can git-ignore `android/app/src/main/jniLibs` instead

Reply:
Done